File: Selenium/Project/CRM_Activity3.java
package Project;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class CRM_Activity3 {

	WebDriver driver;
	WebDriverWait wait;
    
    @BeforeClass
    public void setUp() {
        //Open a browser
        driver = new ChromeDriver();       
        driver.get("http://alchemy.hguy.co/crm");
    }
  
    @Test(priority = 1)
    public void copyrightText() {
    	
    	String textCR = driver.findElement(By.id("admin_options")).getText();
    	System.out.println("The first copyright text is : " + textCR);
    }
    
    
    @AfterClass
    public void tearDown() {
        driver.quit();
    }
}

File: Selenium/Project/CRM_Activity3.py
# Get the copyright text
# Goal: Print the first copyright text in the footer to the console

# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Chrome() as driver:
  
#  Navigate
  driver.get("https://alchemy.hguy.co/crm/")

# Get the first copyright text in the footer
  copyrightText = driver.find_element(By.ID, "admin_options").text
# Print the text to the console.
  print("The text in the Copyright in Footer is : ", copyrightText)

# Close the browser
  driver.quit()